src/elements/content-picker/stories/tests/ContentPicker-visual.stories.js

```diff
@@ -118,30 +118,6 @@ export const customActionButtons = {
     },
 };
 
-export const itemCustomActions = {
```

Comment:
this test is consistently failing, I don't know why. it's likely related to some testing dependency within the storybook packages. it seems that that react-virtualized + blueprint is not playing well

the content picker implementation will be changed in the future to remove react-virtualized so we shouldn't run into this issue again. this test case is already handled here: https://github.com/box/box-ui-elements/blob/463839ee00c901cd9752106ca1a44e7a3906dc0b/src/elements/content-picker/__tests__/ContentPicker.test.tsx#L385-L430